docs/sources/next/using-k6-browser/migrating-to-k6-v0-52.md

```diff
@@ -0,0 +1,309 @@
+---
+title: 'Migrating browser scripts to k6 v0.52'
+description: 'A migration guide to ease the process of transitioning to the new non-experimental k6 browser module version bundled with k6 v0.52'
+weight: 04
+---
+
+# Migrating browser scripts to k6 v0.52
+
+This guide outlines the key changes you will need to make when moving your existing k6 browser test scripts to the _latest_ non-experimental [k6 browser module](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser) version bundled with [k6 0.52](https://github.com/grafana/k6/releases/tag/v0.52.0).
+
+## Key changes
+
+In the latest release of k6, we have introduced the non-experimental k6 browser module under the import `k6/browser`. Migrating to this is a breaking change that will affect _all scripts_ that use the current experimental k6 browser module. The breaking changes are:
+
+1. Converted most of the k6 browser module APIs to asynchronous (async) APIs. That means they will return a `promise` that will `resolve` to a value when the API call succeeds or `reject` when an error occurs.
+2. A side effect of making this async changes is that you will need to use a workaround to work with the k6 [check](http://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6/check/) API.
+3. Finally, it's also worth reiterating that the [group](http://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6/group/) API still doesn't work with async APIs.
+
+If you are interested in the rationale behind this change, refer to the [v0.51 release notes](https://github.com/grafana/k6/releases/tag/v0.51.0).
+
+{{% admonition type="note" %}}
+
+The experimental import (`k6/experimental/browser`) and the corresponding synchronous APIs will be supported up until the 23rd of September 2024.
+
+{{% /admonition %}}
+
+## Migrating to async
+
+To ensure your scripts work with the latest release of the non-experimental k6 browser module, you must:
+
+1. change the import from `k6/experimental/browser` to `k6/browser`.
+2. ensure that all the synchronous (sync) APIs that have been migrated over to async are handled appropriately. In most cases, you will only need to add `await` in front of the API call.
```

Comment:
Should we capitalize the first letter of the items? The other list above is like so.

Reply:
Yep, seems to be how things are done in the [style guide](https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/) 👍 

Changed in a1646a3368d2560501b038a2164918959ea5b193